test/files/problem.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
I didn't end up changing this, just for time reasons. The current file should pretty closely match the HTML that's rendered in Chrome after the Backend response is received. However, the next time I'm in these tests or otherwise reach for `problem.html`, I might go ahead and switch out the HTML file for JSON.